### PR TITLE
Fix: Tests/QUnit - Parents/modules/tests names [ED-2262]

### PR DESCRIPTION
**Pull request by iNewLegend**
~Depends on https://github.com/elementor/elementor/pull/14329~

### CODE REVIEW

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: tests/qunit/tests/assets/dev/js/editor/document/elements/component.spec.js [hunk 1]
```diff
@@ -3,52 +3,50 @@ import ElementsHelper from './helper';
...
-	} );
+	// QUnit.test( 'ResizeColumn', ( assert ) => {
+	// 	//assert.equal( 1, 1, 'Test was skipped.' );
+	//
+	// 	const newSize = 20,
+	// 			eSection = DocumentHelper.createSection( 2 ),
+	// 			eColumn1 = eSection.view.children.findByIndex( 0 ).getContainer(),
+	// 			eColumn2 = eSection.view.children.findByIndex( 1 ).getContainer(),
+	// 			column2NewSize = 100 - newSize;
+	//
```
[the remark this answers is not in the document]
Comment (iNewLegend) [reply tests/qunit/tests/assets/dev/js/editor/document/elements/component.spec.js:23]: @StyleShit @matipojo, the problem with that test that `resizeColumn` is based on a `div` with class name `elementor-container` which created in the backend to fix the issue we need to add a new mechanism to simulate it.

File: tests/screenshotter/backstop.js [hunk 2]
```diff
@@ -12,8 +12,8 @@ const getScenarios = () => {
 			readyEvent: '',
 			readySelector: '',
 			delay: 2000,
+			selectors: [ 'document' ], // With the `document` selector it takes a full page shot.
 			onBeforeScript: `${ __dirname }/scripts/on-before-ready.js`,
```
[the remark this answers is not in the document]
Comment (iNewLegend) [reply tests/screenshotter/backstop.js]: Handled.

File: tests/qunit/tests/core/common/assets/js/api/core/commands.spec.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+				getNamespace() {
+					return namespace;
+				}
+
+				defaultCommands() {
+					return {
+						commandA: () => {
+						},
+					};
+				}
```
[the remark this answers is not in the document]
Comment (iNewLegend) [reply tests/qunit/tests/core/common/assets/js/api/core/commands.spec.js]: Handled.

File: tests/qunit/tests/core/common/assets/js/api/core/commands.spec.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+			};
+
+			$e.components.register( new Component() );
+
+			assert.throws(
+				() => {
+					$e.commands.register( namespace, 'commandA', () => {
+					} );
+				},
```
[the remark this answers is not in the document]
Comment (iNewLegend) [reply tests/qunit/tests/core/common/assets/js/api/core/commands.spec.js]: Handled.

**Review by StyleShit — APPROVED** [review]
LGTM

**Review by Nevoss — CHANGES_REQUESTED** [review]
Change the PR name to adjust to the new convention.
something like: `Fix: Tests/QUnit -  Parents/modules/tests names [ED-2262]`

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull from 3.3.0

**Pull request merged**